src/exampleTypes/ResultTable/TableBody.js

```diff
@@ -13,38 +29,61 @@ let TableBody = ({
   schema,
   Row = 'tr',
   getRowKey = _.get('_id'),
-}) => (
-  <tbody>
-    {!!getResults(node).length &&
-      _.map(
-        x => (
-          <Row
-            key={getRowKey(x)}
-            record={getRecord(x)}
-            {...{ fields, visibleFields, hiddenFields }}
-          >
-            {_.map(
-              ({ field, display = x => x, Cell = 'td' }) => (
-                <Cell key={field}>
-                  {display(_.get(field, getRecord(x)), getRecord(x))}
-                </Cell>
-              ),
-              visibleFields
-            )}
-            {node.showOtherMatches && (
-              <HighlightedColumn
-                {...{
-                  node,
-                  additionalFields: _.result('additionalFields.slice', x),
-                  schema,
-                }}
-              />
-            )}
-          </Row>
-        ),
-        getResults(node)
-      )}
-  </tbody>
-)
+  limitedResults,
+  pageSize,
+}) => {
+  let results = getResults(node)
+  if (limitedResults) {
+    let blankResults = [...Array(pageSize - results.length)].map((_, i) => ({
+      ...results[i % results.length],
+      _id: Math.random(),
+      isBlank: true,
+    }))
+    results = [...results, ...blankResults]
+  }
+  return (
+    <tbody>
+      {!!results.length &&
+        _.map(
+          x => (
+            <Row
+              key={getRowKey(x)}
+              record={getRecord(x)}
+              {...{ fields, visibleFields, hiddenFields }}
+            >
+              {_.map(
+                ({ field, display = x => x, Cell = 'td' }) => (
+                  <Cell key={field}>
+                    {x.isBlank ? (
+                      <span style={{ opacity: 0.2, fontFamily: 'monospace' }}>
+                        {blankResult(
+                          display,
+                          _.get(field, getRecord(x)),
+                          getRecord(x)
+                        )}
+                      </span>
+                    ) : (
+                      display(_.get(field, getRecord(x)), getRecord(x))
+                    )}
```

Comment:
i think you can simplify this by just making `blankResult` handle wrapping in the span and also just making it replace the display function so all you do is swap display instead of duplicating the logic for what gets passed to display

Reply:
Working on it